docs/docs/icicle/colab-instructions.md

```diff
@@ -90,19 +90,48 @@ Now we are ready to clone ICICE repository,
 !git clone https://github.com/ingonyama-zk/icicle.git
 ```
 
-We now can browse the repository and run tests to check the runtime environment:
+We can browse the repository and run tests to check the runtime environment:
 
 ```sh
 !ls -la
-%cd icicle
+%cd /content/icicle
 ```
 
+## Download CUDA backend
+
+First let's create a backend directory
+
+```sh
+%cd /content
+!rm -rf cuda_backend/
+!mkdir cuda_backend
+%cd cuda_backend
+```
+
+Download and extract a backend from [ICICLE released](https://github.com/ingonyama-zk/icicle/releases) backends
+In this example we are using [ICICLE Cuda backend v3.1.0](https://github.com/ingonyama-zk/icicle/releases/download/v3.1.0/icicle_3_1_0-ubuntu22-cuda122.tar.gz)
+
+```sh
+!curl -O -L https://github.com/ingonyama-zk/icicle/releases/download/v3.1.0/icicle_3_1_0-ubuntu22-cuda122.tar.gz
+!tar -xvf icicle_3_1_0-ubuntu22-cuda122.tar.gz
+```
+
+## Setting CUDA backend installation directory
+Point colab to the extracted cuda backend using an [environment variable](https://github.com/ingonyama-zk/icicle/blob/f638e9d3056d2a5d6271a67ba4f63973a2ba2c1a/docs/docs/icicle/getting_started.md#backend-loading)
+
+```sh
+import os
+os.envvar["ICICLE_BACKEND_INSTALL_DIR"] = "/content/cuda_backend/icicle"
+```
+
+## Fun with ICICLE
+
 Let's run a test!
 Navigate to icicle/wrappers/rust/icicle-curves/icicle-bn254 and run cargo test:
 
 ```sh
-%cd wrappers/rust/icicle-curves/icicle-bn254/
-!cargo test --release
+%cd /content/icicle/wrappers/rust/icicle-curves/icicle-bn254/
+!cargo test --release -- ntt
```

Comment:
Why specifically the NTT tests?

Reply:
Only because it relates to the output, and these tests are kind of quick to run just to test that colab is working with GPU